_includes/footer.html

```diff
@@ -37,6 +37,13 @@
               >Tutorials</a
             >
           </li>
+          <li class="my-2">
+            <a
+              class="link-white hover:border-purple-lightest"
+              href="{{ site.integrations }}"
+              >Integrations</a
+            >
```

Comment:
Can we have "Integrations" either below Tutorials or API Docs? It's below Tutorials in navbar and API Docs in footer.